playground/CustomTest.tsx

```diff
@@ -0,0 +1,52 @@
+import { Mermaid } from "./Mermaid.tsx";
+import type { MermaidData } from "./index.tsx";
+
+const CustomTest = ({
+  onChange,
+  mermaidData,
+}: {
+  onChange: (definition: string) => void;
+  mermaidData: MermaidData;
+}) => {
+  const isActiveCustomTest = window.location.hash === "#custom-diagram";
```

Comment:
This is hacky, let's not do this and it also leads to an error when you write a custom test case and refresh the browser due to the type not being found. Let's do it in a react way instead, when clicking on the "Render to Excalidraw" Button you can decide whether it is a custom test case and accordingly pass a boolean variable `isCustom`